benchmarks/common/benchmark_runner.hpp

```diff
@@ -303,22 +304,47 @@ struct BenchmarkRunner {
   ~BenchmarkRunner() {
     benchmark::Shutdown();
   }
-
+    
 private:
   static void run_benchmark(benchmark::State& state, const Options& options,  Gemm gemm_op) {
-    state.counters["runtime_ms"] = 0;
+    auto tflop = ((2.0 * options.m * options.n * options.k * options.l) * 1e-12);
+    auto giga_bytes_transferred = ((options.m * options.k) * sizeof(ElementA) +
```

Comment:
Shouldn't this calculation also include the multiplication with `options.l` to add the bytes moved for all batches?

Reply:
They definitely should 😃 
Thanks

Reply:
Fixed